cpp/ql/src/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -484,6 +499,14 @@ ExprNode convertedExprNode(Expr e) { result.getConvertedExpr() = e }
  */
 ExplicitParameterNode parameterNode(Parameter p) { result.getParameter() = p }
 
+/**
+ * Gets the `Node` corresponding to a definition by reference of the variable
+ * that is passed as unconverted `argument` of a call.
+ */
+DefinitionByReferenceNode definitionByReferenceNodeFromArgument(Expr argument) {
+  result.getArgument() = argument
+}
+
```

Comment:
The body of `definitionByReferenceNodeFromArgument` is identical to `definitionByReferenceNode` on line 479. Is there supposed to be a distinction, or is it just a duplicated predicate?

Reply:
Good catch. I've deprecated `definitionByReferenceNode` since it was IR-only and updated its callers.